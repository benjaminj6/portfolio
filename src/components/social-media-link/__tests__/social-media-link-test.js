/* eslint-env jest */
import { h } from 'preact'
import { shallow } from 'preact-render-spy'
import SocialMediaLink from '..'

test('SocialMediaLink snapshot', () => {
  const tree = shallow(<SocialMediaLink />)
  expect(tree).toMatchSnapshot()
})
