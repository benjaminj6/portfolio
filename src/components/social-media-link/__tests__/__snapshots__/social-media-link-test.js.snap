// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialMediaLink snapshot 1`] = `
preact-render-spy (1 nodes)
-------
<a
  target="_blank"
  class="jsx-3618815995 jsx-3618815995"
>
  <JSXStyle
    styleId="1918456904"
    css=".__jsx-style-dynamic-selector{--fill:var(--gray-dark);display:inline-block;padding:1rem 1.25rem;}.__jsx-style-dynamic-selector:hover,.__jsx-style-dynamic-selector:active,.__jsx-style-dynamic-selector:focus{--local-accent:var(--fill);}"
    dynamic={
      Array [
        "var(--gray-dark)"
      ]
    }
  >
  </JSXStyle>
</a>

`;
